bin/run_tests.sh

```diff
@@ -4,6 +4,13 @@ set -e
 base=$(pwd)
 failed=0
 
+# run snyk tests locally. Skip in bamboo due to authentication token requirement.
```

Comment:
if we also want to run these snyk tests in bamboo, then we have to create a variable for passing the snyk authentication token and the pipeline will have to be run manually which is extra step.
I also noticed there is already a stage thats runs snyk scan in bamboo. 
See https://github.com/nasa/cumulus-orca/blob/develop/bamboo-specs/bamboo.yaml#L96